wandb/integration/keras/keras.py

```diff
@@ -727,18 +727,6 @@ def on_train_end(self, logs=None):
         if self._model_trained_since_last_eval:
             self._attempt_evaluation_log()
 
-    def on_test_begin(self, logs=None):
-        pass
-
-    def on_test_end(self, logs=None):
-        pass
-
-    def on_test_batch_begin(self, batch, logs=None):
-        pass
-
-    def on_test_batch_end(self, batch, logs=None):
-        pass
-
```

Comment:
These were the only errors `ruff` wouldn't autofix; they are redefinitions (identical) from line 660 above.